README.md

```diff
@@ -1,94 +1,160 @@
-# ModSecurity Docker image
+ModSecurity Docker image
+========================
```

Comment:
Since we have more than just one sub-header, we should keep the `#` https://github.github.com/gfm/#atx-heading